src/org/sosy_lab/java_smt/solvers/bitwuzla/BitwuzlaTheoremProver.java

```diff
@@ -246,6 +251,11 @@ protected BitwuzlaModel getEvaluatorWithoutChecks() {
             Collections2.transform(getAssertedFormulas(), creator::extractInfo)));
   }
 
+  @Override
+  protected ShutdownManager getShutdownManagerForProverImpl() throws UnsupportedOperationException {
+    return proverShutdownManager;
+  }
```

Comment:
One could move this method higher up in the hierarchy and just provide it in the abstract super-class.

Reply:
I removed it completely, as we didn't really need it.